### PR TITLE
Remove link to developer console

### PR DESCRIPTION
**Pull request by Martchus**
This link shouldn't be visible when we deploy because it is still experimental.

### CODE REVIEW

File: t/33-developer_mode.t [hunk 1]
```diff
@@ -202,17 +202,8 @@ sub start_worker {
 start_worker;
 OpenQA::Test::FullstackUtils::wait_for_job_running($driver, 'fail on incomplete');
 
-my $developer_console_url;
```
Comment (foursixnine) [inline t/33-developer_mode.t]: I think you still need this line :P

**Comment by coolo** [comment]
the test failure is real :)

**Comment by Martchus** [comment]
@coolo Yes. The web socket connection just timed out too early. For now, I've just increased the inactivity timeout and the check for 'paused' seems to pass now:

```
ok 20 - job is running
# Subtest: wait until developer console becomes available
 - waiting for worker to propagate URL for os-autoinst cmd srv
 - waiting for worker to propagate URL for os-autoinst cmd srv
 - waiting for worker to propagate URL for os-autoinst cmd srv
 - waiting for worker to propagate URL for os-autoinst cmd srv
 - waiting for worker to propagate URL for os-autoinst cmd srv
    ok 1 - os-autoinst cmd srv available after 5 seconds
    ok 2 - no errors or warnings on javascript console, waiting for connection opened
    ok 3 - found connection opened
    1..3
ok 21 - wait until developer console becomes available
# Subtest: pause at certain test
    ok 1 - no errors or warnings on javascript console, waiting for response to set_pause_at_test
    ok 2 - found response to set_pause_at_test
    ok 3 - no errors or warnings on javascript console, waiting for paused
    ok 4 - no errors or warnings on javascript console, waiting for resume
    1..4
ok 22 - pause at certain test
```

However, it now fails later on:  
```
not ok 24 - test 1 is passed
#   Failed test 'test 1 is passed'
#   at /opt/testing_area/openqa/t/lib/OpenQA/Test/FullstackUtils.pm line 85.
#                   'State: running started 2 minutes ago
# Assigned worker: 2beaa8807e04:1'
#     doesn't match '(?^:Result: passed)'
not ok 25 - log file generated
#   Failed test 'log file generated'
#   at t/full-stack.t line 226.
not ok 26 - image of hdd uploaded
#   Failed test 'image of hdd uploaded'
#   at t/full-stack.t line 227.
Use of uninitialized value in subroutine entry at t/full-stack.t line 228 (#1)
    (W uninitialized) An undefined value was used as if it were already
    defined.  It was interpreted as a "" or a 0, but maybe it was a mistake.
    To suppress this warning assign a defined value to your variables.
    
    To help you figure out what was undefined, perl will try to tell you
    the name of the variable (if any) that was undefined.  In some cases
    it cannot do this, so it also tells you what operation you used the
    undefined value in.  Note, however, that perl optimizes your program
    anid the operation displayed in the warning may not necessarily appear
    literally in your program.  For example, "that $foo" is usually
    optimized into "that " . $foo, and the warning will refer to the
    concatenation (.) operator, even though there is no . in
    your program.
    
not ok 27 - exported image has correct permissions (420 -> 0644)
#   Failed test 'exported image has correct permissions (420 -> 0644)'
#   at t/full-stack.t line 229.
#          got: '0'
#     expected: '420'
```

The UI tests fail in a related way:  
```
# Subtest: New needle instantly visible after reloading needle editor
    ok 1 - tests/99946 followed
    ok 2 - warning about new needle displayed
    ok 3 - needle appears in tags_select selection
    ok 4 - needle title in tags_select selection correct
    ok 5 - needle appears in image_select selection
    ok 6 - needle title in image_select selection correct
    ok 7 - tag_inst-timezone not checked anymore
    ok 8 - tag_inst-timezone checked again via new needle
    ok 9 - screenshot shown by default
    not ok 10 - new needle image shown
    not ok 11 - new needle image shown
    not ok 12 - new needle image shown
    1..12
not ok 7 - New needle instantly visible after reloading needle editor
```

I don't think it is related to this change (I basically just removed the link to the developer console here).

**Comment by Martchus** [comment]
The test failure is likely related to: https://github.com/os-autoinst/openQA/pull/1690/files#diff-a8857daf563deea1776f5c009cd73a32L120

**Comment by coolo** [comment]
interesting theory but then master would show the same failures, no?

**Comment by Martchus** [comment]
I wasn't aware that the tests pass on master since I saw other PR failing because of the ws timeout. But it seems that the timeout was only exceeded sometimes.

Nevertheless, I found the bug in my code triggering the errors mentioned above. But the fullstack test still fails:

```
not ok 47 - Test 3 is scheduled
#   Failed test 'Test 3 is scheduled'
#   at /opt/testing_area/openqa/t/lib/OpenQA/Test/FullstackUtils.pm line 85.
#                   'State: assigned
# Clone of 2
# Assigned worker: 5a44f9dd89fc:1'
#     doesn't match '(?^:State: scheduled)'
Uncaught exception from user code:
	Error while executing command: no such element: Unable to locate element: {"method":"css selector","selector":"[id='cancel_running']"}
	  (Session info: headless chrome=66.0.3359.170)
	  (Driver info: chromedriver=2.36 (0),platform=Linux 4.4.0-101-generic x86_64) at /usr/lib/perl5/vendor_perl/5.18.2/Selenium/Remote/Driver.pm line 388.
	 at /usr/lib/perl5/vendor_perl/5.18.2/Selenium/Remote/Driver.pm line 345.
	Selenium::Remote::Driver::find_element('Test::Selenium::Chrome=HASH(0xd600fe0)', 'cancel_running', 'id') called at /usr/lib/perl5/vendor_perl/5.18.2/Test/Selenium/Remote/Driver.pm line 345
	Test::Selenium::Remote::Driver::_find_element_with_action('Test::Selenium::Chrome=HASH(0xd600fe0)', 'click_ok', 'cancel_running', 'id', 'Caught cancel') called at /usr/lib/perl5/vendor_perl/5.18.2/Test/Selenium/Remote/Driver.pm line 400
	Test::Selenium::Remote::Driver::click_element_ok('Test::Selenium::Chrome=HASH(0xd600fe0)', 'cancel_running', 'id', 'Caught cancel') called at t/full-stack.t line 270
```

**Comment by coolo** [comment]
this assigned vs scheduled is a long existant problem. It normally goes away on reschedule :(

**Comment by coolo** [comment]
(I asked @Martchus to merge)

**Pull request merged**